thatkitebot/cogs/electronics.py

```diff
@@ -137,6 +138,8 @@ def calculate_lm317(b):
     r1 = si_prefix.si_parse(b["r1"])
     r2 = si_prefix.si_parse(b["r2"])
     vout = si_prefix.si_format(1.25 * (1 + (r2/r1)))
+    if vin - vout > 40 or vin - vout < 3:
+        raise ValueError("In Out voltage out of Range") # Input-to-output differential voltage out of range
```

Comment:
Using ValueError here would not really work since the bot already raises a ValueError when the input voltage is too high and will not be able to differentiate between the two. Custom Exceptions would be a better approach (like InputTooHighException or something like that)

Reply:
Added custom exception class